src/opnsense/www/js/chart.js

```diff
@@ -16106,7 +16106,7 @@ core_controller.Title = plugins.title._element;
 core_controller.pluginService = core_controller.plugins;
 
 /**
- * Provided for backward compatibility, inheriting from Chart.PlugingBase has no
+ * Provided for backward compatibility, inheriting from Chart.PluginBase has no
```

Comment:
This entire file puzzles me, since apparently I made a PR to that project in ~2016... But I haven't investigated closely.

I'm mostly curious as to why a version that would have been from shortly after my PR would still be in the wild...